.baseline/checkstyle/checkstyle.xml

```diff
@@ -122,7 +122,8 @@
                 org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName.*,
                 org.apache.spark.sql.functions.*,
                 org.apache.spark.sql.connector.iceberg.write.RowLevelOperation.Command.*,
-                org.junit.Assert.*"/>
+                org.junit.Assert.*,
+                org.assertj.core.api.Assertions.*"/>
```

Comment:
I don't think we need to change this.

Reply:
Hi Ryan - without the above change, the `checkstyleIntegration` task fails with the following error:
```
[ant:checkstyle] [ERROR] /<redacted>/upstream-iceberg/azure/src/integration/java/org/apache/iceberg/azure/blob/TestAzureBlobOutputStream.java:38:46: Using a static member import should be avoided - org.assertj.core.api.Assertions.assertThat. [AvoidStaticImport]

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':iceberg-azure:checkstyleIntegration'.
> Checkstyle rule violations were found. See the report at: file:///<redacted>/upstream-iceberg/azure/build/reports/checkstyle/integration.html
  Checkstyle files with violations: 5
  Checkstyle violations by severity: [error:12]
```

Reply:
Yes, you shouldn't use a static import for those methods.

Reply:
Ack, 
will make the necessary modifications.